pkgs/development/interpreters/ruby/build-ruby-gem/gem-post-build.rb

```diff
@@ -64,8 +64,7 @@
 # this file is here to facilitate running it.
 #
 
-gem_path = ENV["GEM_PATH"]
-ENV["GEM_PATH"] = "\#{gem_path}\#{":" unless gem_path.nil? || gem_path.empty?}#{(gem_path+[gem_home]).join(":")}"
+Gem.use_paths "#{gem_home}", #{gem_path.to_s}
 
 require 'rubygems'
```

Comment:
It's not necessary anymore, ruby has been bundling rubygems for a long time now and is just available by default. If you run `ruby -e "p(require 'rubygems')"` you will get `false` which is what is returned when a feature is already loaded.